modules/collect/src/main/java/com/opengamma/collect/io/PropertySet.java

```diff
@@ -109,40 +152,39 @@ public String getValue(String key) {
    * A key-values instance may contain multiple values for each key.
    * This method returns that list of values.
    * The iteration order of the map matches that of the input data.
+   * The returned list may be empty.
    * 
    * @param key  the key name
    * @return the list of values associated with the key
-   * @throws IllegalArgumentException if the key does not exist
    */
   public ImmutableList<String> getValueList(String key) {
     ArgChecker.notNull(key, "key");
-    if (contains(key) == false) {
-      throw new IllegalArgumentException("Unknown key: " + key);
-    }
-    return keyValueMap.get(key);
+    return MoreObjects.firstNonNull(keyValueMap.get(key), ImmutableList.<String>of());
   }
 
   //-------------------------------------------------------------------------
   /**
-   * Gets all the keys of this property set.
+   * Combines this property set with another.
    * <p>
-   * The iteration order of the map matches that of the input data.
+   * The specified property set takes precedence.
```

Comment:
This is inconsistent with other ``combinedWith`` methods where the receiver data takes priority over the argument data.